bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/BagVerifierWorkerService.scala

```diff
@@ -122,11 +122,10 @@ class BagVerifierWorkerService(
     }
     f"""$verificationStatus verification
        |of ${bagRequest.bagLocation.completePath}
-       |completed in ${bagVerification.duration.getSeconds}s
+       |completed in ${bagVerification.duration.getOrElse("")}
```

Comment:
I think this would be better we were really explicit about what's happening here. How about:

```suggestion
       |completed in ${bagVerification.duration.getOrElse("<NOTCOMPLETE>")}
```

Reply:
:+1: